src/paperoni/utils.py

```diff
@@ -178,6 +178,22 @@ def associate(l1, l2, key, threshold=0):
     return [(x1, mapping.get(i1, None)) for i1, x1 in el1]
 
 
+def normalize_institution(institution: str) -> str:
+    return unidecode(institution).lower()
+
+
+def normalize_name(name: str) -> str:
+    return unidecode(name).lower()
+
+
+def normalize_title(title: str) -> str:
+    return unidecode(title).lower()
```

Comment:
Use `plainify` that is defined in the same file, which calls unidecode but also removes some characters like . or - that constitute the majority of minor title differences.

I'm tempted to also remove all spaces to handle semantic scholar's messed up "T ITLES L IKE T HIS".